src/app/view/components/_common/dot-icon/dot-icon.component.scss

```diff
@@ -0,0 +1,15 @@
+@import 'variables';
+
+.material-icons {
+    color: rgba(126, 122, 134, 1);
+    font-size: $font-size-xx-large;
```

Comment:
Shouldn't we create a font size variable for the icons? or just set it here directly?